plant-palace/src/main/java/mk/ukim/finki/uiktp/plantpalace/model/dto/RegisterDto.java

```diff
@@ -0,0 +1,13 @@
+package mk.ukim.finki.uiktp.plantpalace.model.dto;
+
+import lombok.Data;
+import mk.ukim.finki.uiktp.plantpalace.model.enumerations.Role;
+
+@Data
+public class RegisterDto {
+    private String username;
+    private String password;
+    private String repeatPasword;
```

Comment:
This property is not needed in the backend, as the password repeat check will be already done in the frontend, so it can be deleted.